interface-definitions/snmp.xml.in

```diff
@@ -123,18 +123,35 @@
           </leafNode>
           <leafNode name="oid-enable">
             <properties>
-              <help>Enable specific OIDs</help>
+              <help>Enable specific OIDs that by default are disable</help>
               <completionHelp>
-                <list>route-table</list>
```

Comment:
It is still used by https://github.com/vyos/vyos-1x/blob/65b104d6e0608e229aa36d948fabddaf3f4a0a8b/data/templates/snmp/override.conf.j2
Also, if we remove this from here, we should create a migration to `ip-cidr-route-table` (or another option with a similar meaning).